docs/drill4net/support-matrix.mdx

```diff
@@ -0,0 +1,54 @@
+﻿---
+id: support-matrix
+title: Support Matrix
+---
+import { Image } from '@components';
+import { Info } from '@components';
+import { Table } from '../../src/components'
+
+## Testing Frameworks
+
+<Table>
+
+| Framework |      Supported versions     |
+|:---------:|:---------------------------:|
+| NUnit     |	    - 3.13.3			  |
+| XUnit     | 2.4.2 - 4.13.2              |
+| Selenium  |	    - 4.12.4			  |
+| Playwright|		- 1.32.0			  |
```

Comment:
Please provide links to frameworks in question to avoid any misunderstanding
Just make the framework name a link - e.g. `[NUnit](https://link-to-nunit.com)`